httpclient/src/main/java/com/okta/sdk/impl/http/httpclient/HttpClientRequestExecutor.java

```diff
@@ -81,7 +84,7 @@
     /**
      * Maximum exponential back-off time before retrying a request
      */
-    private static final int MAX_BACKOFF_IN_MILLISECONDS = 20 * 1000;
+    private static final int DEFAULT_MAX_BACKOFF_IN_MILLISECONDS = 20 * 1000;
 
     private static final int DEFAULT_MAX_RETRIES = 4;
```

Comment:
Should be 2 per our spec?

Reply:
The current implementation uses 4, for backwards compatibility